frontend/src/scenes/web-analytics/WebDashboard.tsx

```diff
@@ -12,22 +12,17 @@ import { DateFilter } from 'lib/components/DateFilter/DateFilter'
 
 const PercentageCell: QueryContextColumnComponent = ({ value }) => {
     if (typeof value === 'number') {
-        return (
-            <div className="w-full text-right">
-                <span className="flex-1 text-right">{`${(value * 100).toFixed(1)}%`}</span>
-            </div>
-        )
+        return <span>{`${(value * 100).toFixed(1)}%`}</span>
     } else {
         return null
     }
 }
 
+// format numbers according to the user's locale
+const numberFormatter = new Intl.NumberFormat()
```

Comment:
I think it makes sense to construct  `Intl.NumberFormat()` if we passed some options we want to be used commonly here, but if we just want to use the user locale's rendering, it's simpler to call `value.toLocaleString()`